grafana/scylla-dash-per-server.master.template.json

```diff
@@ -127,6 +131,39 @@
                             }
                         ],
                         "title": "Total Storage"
+                    },
+
+                    {
+                        "class": "ns_panel",
+                        "pointradius": 1,
+                        "targets": [
+                            {
+                                "expr": "sum(irate(scylla_scheduler_time_spent_on_task_quota_violations_ns{group=\"main\", instance=~\"[[node]]\", shard=~\"[[shard]]\"}[30s])) by ([[by]])",
```

Comment:
the violation graph shouldn't specify the group. Let me guys know if the rest is okay and I will fix that for next version

Reply:
@amnonh let me know what you think